tests/DocumentationExamplesTest.php

```diff
@@ -1788,6 +1809,7 @@ public function testWithTransactionExample(): void
         $this->skipIfTransactionsAreNotSupported();
 
         $uriString = static::getUri(true);
+        $this->dbNames = ['mydb1', 'mydb2'];
```

Comment:
Per my earlier suggestion about registering databases/collections and dropping them before a test, I do think it's wise to keep that logic _after_ stuff like `skipIfTransactionsAreNotSupported()` to avoid unnecessary work.

Based on where you placed `$this->dbNames = [...]` here I assume you may have already had that in mind.